Automattic/jetpack
# WAF: Update brute force shared functions to namespaced class

#### Description
Moves all functions used in both the brute force module and its related JSON endpoints to a dedicated namespaced class.

#### Proposed changes:

* Updates the current `shared-functions.php` functions to static methods of a `Brute_Force_Protection_Shared_Functions` class
* Updates the current `transient-cleanup.php` functions to static methods of a `Brute_Force_Protection_Transient_Cleanup` class
* Updates all internal and external code reliant on these functions accordingly

### Other information:


## Jetpack product discussion


-1201069996155224-as-1203812254177715

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Start up a Jurassic Ninja instance using the beta plugin on this Jetpack branch
* Enable Jetpack and connect your user
* Verify that Brute force protection is enabled and is functioning as expected
* Modify and save settings
* Ensure the math auth displays after several failed attempts
* Ensure the hard block notice displays after an appropriate number of failed attempts and that the recovery process succeeds
